tools/cellxgene_census_builder/src/cellxgene_census_builder/data_copy.py

```diff
@@ -21,13 +21,30 @@ def sync_to_S3(from_path: Union[str, pathlib.PosixPath], to_path: str, dryrun: b
     if not to_path.startswith("s3://"):
         raise ValueError(f"S3_path argument does not appear to be an S3 path: {to_path}")
 
-    cmd = ["aws", "s3", "sync", from_path.as_posix(), to_path, "--no-progress"]
+    _sync_to_S3(from_path.as_posix(), to_path, dryrun)
+
+
+def sync_to_S3_remote(from_path: str, to_path: str, dryrun: bool = False) -> None:
+    """Copy (sync) between two S3 locations.
+
+    Equivalent of `aws s3 sync S3_path_source S3_path_dst`.
+
+    Raises upon error.
+    """
+    if not from_path.startswith("s3://") or not to_path.startswith("s3://"):
+        raise ValueError(f"S3_path argument does not appear to be an S3 path: {to_path}")
+
+    _sync_to_S3(from_path, to_path, dryrun)
+
+
+def _sync_to_S3(from_path: str, to_path: str, dryrun: bool = False) -> None:
+    cmd = ["aws", "s3", "sync", from_path, to_path, "--no-progress"]
     if dryrun:
         cmd += ["--dryrun"]
 
     returncode = -1
     try:
-        _log_it(f"Starting copy {from_path.as_posix()} -> {to_path}", dryrun)
+        _log_it(f"Starting copy {from_path} -> {to_path}", dryrun)
         with subprocess.Popen(cmd, stderr=subprocess.STDOUT, stdout=subprocess.PIPE, text=True) as proc:
             print(proc.returncode)
```

Comment:
debugging code?

Reply:
The code was already there 🤷 I think it doesn't hurt.

Reply:
but cleanup is goodness :-)   I'll take the blame for the garbage!

Reply:
I'll get rid of it then.

Reply:
I like the logging - it was only the print I was maligning :-)